src/js/va-letters/reducers/index.js

```diff
@@ -1 +1,27 @@
-export default {};
+import set from 'lodash/fp/set';
+
+const initialState = {
+  letters: [],
+  destination: {},
+  available: false
+};
+
+function letters(state = initialState, action) {
+  switch (action.type) {
+    case 'GET_LETTERS_SUCCESS':
+      return {
+        ...state,
```

Comment:
I'm not familiar with this syntax or how this works. Why just `return` these values instead of setting them within `state` as you do below with the `available` property? Are the values for `letters` and `destination` being set in `state` as well here?